include/swift/SIL/SILConstants.h

```diff
@@ -397,21 +397,19 @@ struct SymbolicValueMemoryObject {
   void setValue(SymbolicValue newValue) { value = newValue; }
 
   /// Create a new memory object whose overall type is as specified.
-  static SymbolicValueMemoryObject *create(Type type,
-                                           llvm::BumpPtrAllocator &allocator);
   static SymbolicValueMemoryObject *create(Type type, SymbolicValue value,
+                                           llvm::BumpPtrAllocator &allocator);
+  static SymbolicValueMemoryObject *create(Type type,
```

Comment:
do we still need the second form of create()?

Reply:
Literally no, because it is just sugar for the other one :-), but it is fairly commonly used, e.g. by alloc_stack and other things that allocate uninitialized memory.  I'll see if it makes sense to eliminate in my next patch, which starts doing more interesting things now that we're back to the status quo working with the new representations.